controllers/host/platformnetworks.go

```diff
@@ -710,6 +732,16 @@ func (r *HostReconciler) UpdateAddrPoolReconciliationStatus(
 			addrpool_instance.Status.Reconciled = is_reconciled
 		}
 		addrpool_instance.Status.InSync = is_reconciled
+	} else if network_instance.Status.DeploymentScope == cloudManager.ScopePrincipal {
```

Comment:
As the logic in the bootstrap and principal is identical, can you merge them?

Reply:
Logic is not the same, for bootstrap scope, `Reconciled` will be set to true and `InSync` based on reconciliation.
In case of principal scope, `InSync` will be set to true and `Reconciled` based on reconciliation.

Essentially the inverse of it.

Reply:
I saw the comments about manipulating the values for avoiding alarms, but avoiding alarms especially the 260.002 is not necessary if the alarm is transitory. Can we simplify the logic here if temporary alarm is acceptable?

Reply:
Done, updated such that `InSync` follows `Reconciled` during Day-2.